TEBOWIT.txt

```diff
@@ -1,46 +0,0 @@
-T alk
-what will be return if I have a empty dictionary?  exception
```

Comment:
Excellent question - make sure to double-check your implementation against the "answers" from your interviewer!